### PR TITLE
Cricut Voron Logo - had to shorten the filenames because they were toooooooo long

### PR DESCRIPTION
**Pull request by GadgetAngel**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### How was it tested? 
yes, on my branch of the repository

#### Any background context you want to provide?
Voron user's were complaining about the filename length so I shorten the filenames and fixed the URLs

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -114,6 +114,8 @@ like so:
...
 |  | [Bed Support Extrusions Alignment Tools](./GadgetAngel/Bed_Support_Alignment_Tools) |The bed support alignment tools will assist in helping align the "bed support extrusions" to the frame. When the alignment tools are placed in all four corners of the Voron Printer's frame the "bed support extrusions" will butt up against these tools so that you can screw the "bed support extrusions" down to the printer's frame in the correct position.|  :x: | :x: | :heavy_check_mark: |  :x: | :x: |
+|  | [Bigtreetech Octopus Pro V1.0 Color PIN Diagram](./GadgetAngel/Bigtreetech_Octopus_Pro_V1.0_Color_PIN_Diagram) |A color PIN Diagram in PDF file format (and a JPG file format) for the Octopus Pro V1.0 board that has been colorized so that the user can easily identify which endstops belong to which motors. Also, this PDF file labels all connectors instead of using "J" names. URL links have also been embedded in the PDF file so that the user can find additional information.  | :x: | :heavy_check_mark: | :heavy_check_mark: | :x: | :heavy_check_mark: |
```
Comment (KiloQubit) [inline printer_mods/README.md:117]: These two lines about Octopus diagrams should not be part of this PR.

Comment (GadgetAngel) [reply printer_mods/README.md]: Sorry about that I forgot to delete those after I deleted the Color PIN Diagrams.  Thanks for catching that!  I fixed it and pushed a new update.

**Review by KiloQubit — APPROVED** [review]
Thanks for the update!
